File: tuna-job/src/main/scala/com/yhd/arch/tuna/linktree/dto/NodeInfo.scala
package com.yhd.arch.tuna.linktree.dto

/**
  * 节点信息
  * Created by  root on 10/14/16.
  */
class NodeInfo(poolId:String,
               methodName:String,
               serviceName:String,
               serviceGroups:String,
               serviceVersion:String) {

  def this(nodeName:String,
           methodName:String
          )={
    this(nodeName,methodName,null,null,null)
  }
  def getPoolId():String={
    poolId
  }


  def getServiceName():String={
    serviceName
  }

  def getServiceGroups():String={
    serviceGroups
  }

  def getServiceVersion():String={
    serviceVersion
  }
  def getMethodName():String={
    methodName
  }
}

File: tuna-common/src/main/java/com/yhd/arch/tuna/link/dto/NodeInfo.java
package com.yhd.arch.tuna.link.dto;

/**
 * Created by root on 2/13/17.
 */
public class NodeInfo {
	private String serviceGroups;

	private String methodName;

	private String poolId;

	private String serviceName;

	private String version;

	public void setServiceGroups(String groupNames) {
		this.serviceGroups = groupNames;
	}

	public String getServiceGroups() {
		return serviceGroups;
	}

	public void setMethodName(String methodName) {
		this.methodName = methodName;
	}

	public String getMethodName() {
		return methodName;
	}

	public void setPoolId(String poolId) {
		this.poolId = poolId;
	}

	public String getPoolId() {
		return poolId;
	}

	public void setServiceName(String serviceName) {
		this.serviceName = serviceName;
	}

	public String getServiceName() {
		return serviceName;
	}

	public void setVersion(String version) {
		this.version = version;
	}

	public String getVersion() {
		return version;
	}
}
